.circleci/config.yml

```diff
@@ -156,6 +156,29 @@ jobs:
       - store_artifacts:
           path: integration-tests/screenshots
 
+  upload_test_data_to_s3:
+    executor:
+      name: hmpps/default_small
+    steps:
+      - checkout
+      - hmpps/k8s_setup
+      - hmpps/install_aws_cli
+      - run:
+          name: Upload test data to S3 bucket
+          command: |
+            S3_BUCKET_NAME=$(kubectl --namespace="${KUBE_ENV_NAMESPACE}" get secret analytical-platform-s3-bucket-output -o go-template='{{.data.bucket_name|base64decode}}')
+            export AWS_ACCESS_KEY_ID=$(kubectl --namespace="${KUBE_ENV_NAMESPACE}" get secret analytical-platform-s3-bucket-output -o go-template='{{.data.access_key_id|base64decode}}')
+            export AWS_SECRET_ACCESS_KEY=$(kubectl --namespace="${KUBE_ENV_NAMESPACE}" get secret analytical-platform-s3-bucket-output -o go-template='{{.data.secret_access_key|base64decode}}')
```

Comment:
uses `kubectl` to load S3 bucket details directly into environment variables. bucket name is only used as a substitution variable so does not need to be exported, but AWS credentials must be exported to be inherited by `aws` process.